source/transactions/tests/error-labels.yml

```diff
@@ -698,7 +698,7 @@ tests:
         object: session0
         result:
           errorLabelsContain: ["UnknownTransactionCommitResult"]
-          errorLabelsOmit: ["TransientTransactionError"]
+          errorLabelsOmit: ["RetryableWriteError", "TransientTransactionError"]
```

Comment:
For the benefit of other reviewers, I'll point out that these tests correspond to https://github.com/mongodb/specifications/blob/master/source/transactions/transactions.rst#unknowntransactioncommitresult. Specifically:

> These errors codes mean that the provided write concern is not valid and therefore a retry attempt would fail with the same error.

That explains why we use the UnknownTransactionCommitResult label but _not_ RetryableWriteError.